README.md

```diff
@@ -12,6 +12,13 @@ Make sure the global composer project is configured properly with your shell.
 Notably, the composer global bin directory needs to be in your path.
 See https://getcomposer.org/doc/03-cli.md#global for instructions on how todo that.
 
+## Adding composer global bin to path on Ubuntu
+
+You should add to your ~/.profile 
+```
+export PATH="$PATH:$COMPOSER_HOME/vendor/bin"
```

Comment:
I suppose the majority of composer users does not have the `$COMPOSER_HOME` environment variable set so this line will not work there.

Also `.profile` is not read by some shells like zsh.